### PR TITLE
Use Apple's version of libffi 

### PR DESCRIPTION
**Pull request by CuriousTommy**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -313,3 +313,6 @@
 [submodule "src/external/cups"]
 	path = src/external/cups
 	url = ../darling-cups.git
+[submodule "src/external/libffi"]
+	path = src/external/libffi
+	url = ../darling-libffi
```
Comment (bugaevc) [inline .gitmodules]: `.git`

Comment (CuriousTommy) [reply .gitmodules]: Fixed

**Comment by bugaevc** [comment]
LGTM otherwise 🙂

**Comment by CuriousTommy** [comment]
@bugaevc Can I merge it or not yet?

**Comment by bugaevc** [comment]
I usually just put my LGTM and leave it to some other team member to review & merge contributions in the parts of the project I'm not responsible for. But I guess in this case the other team member is you, so, go ahead :smile:

**Pull request merged**